web/scripts/start.sh

```diff
@@ -15,6 +15,7 @@ fi
 
 python3 manage.py migrate --noinput
 python3 manage.py collectstatic --noinput
+python3 manage.py graphql_schema --schema impact.graphql.schema.schema
```

Comment:
I'm not sure what this line is doing. Could you clarify what this line does?

Reply:
@MattMSumner same as `gql:schema` but server side.